packages/llvm.rb

```diff
@@ -1,179 +1,92 @@
 require 'package'
 
 class Llvm < Package
-  description 'The LLVM Project is a collection of modular and reusable compiler and toolchain technologies. The optional packages clang, lld, lldb, polly, compiler-rt, libcxx, libcxxabi, and openmp are included.'
-  homepage 'http://llvm.org/'
-  @_ver = '11.1.0'
-  version @_ver
-  license 'Apache-2.0-with-LLVM-exceptions, UoI-NCSA, BSD, public-domain, rc, Apache-2.0 and MIT'
+  description 'The LLVM Core libraries provide a modern source and target-independent optimizer, along with code generation support.'
+  homepage 'https://llvm.org/'
+  version '11.1.0'
+  license 'Apache-2.0-with-llvm-exceptions'
   compatibility 'all'
-  source_url "https://github.com/llvm/llvm-project/archive/llvmorg-#{@_ver}.tar.gz"
-  source_sha256 '53a0719f3f4b0388013cfffd7b10c7d5682eece1929a9553c722348d1f866e79'
-
-  binary_url({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/llvm-11.1.0-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/llvm-11.1.0-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/llvm-11.1.0-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/llvm-11.1.0-chromeos-x86_64.tar.xz'
-  })
-  binary_sha256({
-    aarch64: 'ba0520a8d5bfa90efb3eb13a530c8d4052f9e7572aa30dd42dfc624b88a0251e',
-     armv7l: 'ba0520a8d5bfa90efb3eb13a530c8d4052f9e7572aa30dd42dfc624b88a0251e',
-       i686: '3a514f96c846d777c838fef6c585d2278abf2a586bfaeacfd3e2bccc9ffce838',
-     x86_64: '5b9e01c2411ea174293fc1d33b2dbf91f7001eb3cc3eda2e849b80f6f0dcb3c8'
-  })
+  source_url 'https://github.com/llvm/llvm-project/releases/download/llvmorg-11.0.1/llvm-11.0.1.src.tar.xz'
+  source_sha256 'ccd87c254b6aebc5077e4e6977d08d4be888e7eb672c6630a26a15d58b59b528'
 
+  depends_on 'zlibpkg'
+  depends_on 'ncurses'
+  depends_on 'libffi'
+  depends_on 'libxml2'
   depends_on 'ocaml' => :build
+  depends_on 'cmake' => :build
+  depends_on 'perl' => :build
+  depends_on 'pkgconfig' => :build
+  depends_on 'binutils' => :build
+  depends_on 'linuxheaders' => :build
+  depends_on 'python3' => :build
   depends_on 'pygments' => :build
-  depends_on 'ccache' => :build
-  # depends_on 'llvm_stage1' => :build # Not needed with a functional
-  # prior llvm 11 build.
 
   case ARCH
   when 'aarch64', 'armv7l'
     # LLVM_TARGETS_TO_BUILD = 'ARM;AArch64;AMDGPU'
     LLVM_TARGETS_TO_BUILD = 'all'.freeze
-    @ARCH_C_FLAGS = "-fPIC -march=armv7-a -mfloat-abi=hard -ccc-gcc-name #{CREW_BUILD}"
-    @ARCH_CXX_FLAGS = "-fPIC -march=armv7-a -mfloat-abi=hard -ccc-gcc-name #{CREW_BUILD}"
+    @ARCH_C_FLAGS = "-fPIC  -pipe -march=armv7-a -mfloat-abi=hard -ccc-gcc-name #{CREW_TGT}"
+    @ARCH_CXX_FLAGS = "-fPIC -pipe -march=armv7-a -mfloat-abi=hard -ccc-gcc-name #{CREW_TGT}"
     @ARCH_LDFLAGS = ''
-    @ARCH_LTO_LDFLAGS = "#{@ARCH_LDFLAGS} -flto=thin"
+    @ARCH_LTO_LDFLAGS = "#{@ARCH_LDFLAGS} -flto=auto"
```

Comment:
`-flto=auto` only works for gcc.  This might have to be built with clang, which will require `-flto=thin`

Reply:
it seems to build fine with gcc, and gcc produces faster binaries than clang.